components/mentorDashboard/Bookings.js

```diff
@@ -81,7 +82,13 @@ const Bookings = ({ setLoadingState, setErrorState }) => {
 
       {/* sessions list */}
       <div className="tw-mt-8">
-        <ul className="tw-bg-primary-100 tw-text-white  tw-flex tw-items-center tw-gap-6 tw-p-6 tw-border tw-border-b-0 tw-rounded-t">
+        {sessions ? (
+          <TableComponent sessions={sessions} activeTab={activeTab} />
+        ) : (
+          <Spinner />
+        )}
+
+        {/* <ul className="tw-bg-primary-100 tw-text-white  tw-flex tw-items-center tw-gap-6 tw-p-6 tw-border tw-border-b-0 tw-rounded-t">
```

Comment:
can you same implement for user dashboard to use this table component in case of bookings please!